### PR TITLE
Lesson 18

### PR DESCRIPTION
**Pull request by dero5001**
Please check my answers.
Thank you in advance.

### CODE REVIEW

File: Lesson_18/Tusk_1.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Bot:
+    def __init__(self, name):
+        self.name = name
+
+    def say_name(self):
+        print(self.name)
+
+    def send_message(self, message):
+        self.message = message
```
Comment (klymenok) [inline Lesson_18/Tusk_1.py:9]: You don't need to create a new attribute here.

File: Lesson_18/Tusk_4.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class User:
+    def __init__(self, name):
+        self.name = name.upper()
```
Comment (klymenok) [inline Lesson_18/Tusk_4.py:3]: You transform the initial value here, but the goal is to keep initial value as it is and transform it during comparison:
```
return self.name.upper() == other.upper()
```

**Review by klymenok — APPROVED** [review]
Looks good, but some improvements can be made

File: Lesson_18/Tusk_4.py (hunk 2 above)
Comment (dero5001) [reply Lesson_18/Tusk_4.py:3]: If I do like this appears an error:
Traceback (most recent call last):
  File "C:\Users\Роман\PycharmProjects\robot_dreams_tusks\Lesson_18\Tusk_4.py", line 12, in <module>
    print(user_1 == user_2)
  File "C:\Users\Роман\PycharmProjects\robot_dreams_tusks\Lesson_18\Tusk_4.py", line 6, in __eq__
    return self.name.upper() == other.upper()
AttributeError: 'User' object has no attribute 'upper'

**Pull request merged**